### DISCUSSION TITLE
nom tables api

### DISCUSSION THREAD

**Discussion by tom6450** [dbarzin/mercator#185] [General]
Bonjour,

Dans l'api les tables sont notées avec un tiret tandis que dans la base de données elles sont écrites avec un underscore. Cela porte à confusion.
Pourriez-vous me confirmer cette observation?

API "-"
https://dbarzin.github.io/mercator/api/#les-apis

API "_"
https://dbarzin.github.io/mercator/model/

**Comment by dbarzin** [comment]
Oui, effectivement, c'est une convention de nommage. Cela ne devrait pas porter à confusion car c'est comme cela pour toutes les API. 
Vous souhaitez remplacer le tire par un underscore ?

**Reply by tom6450** [reply]
Merci de votre réponse . Je n'étais pas au courant pour la convention, Cela m'avait légèrement induit en erreur lors de ma première requête, c'est pour cela que je me suis permis de vous en faire part. Si c'est la convention, alors je me plie à celle-ci.
